docs/core/compatibility/containers/8.0/aspnet-port.md

```diff
@@ -7,27 +7,62 @@ ms.date: 07/12/2023
 
 Prior to .NET 8, you could run a container expecting port 80 to be the default port and be able to access the running app.
 
-For example, running the following command allowed you to access the app locally at port 9999, which is mapped to port 80 in the container:
+For example, running the following command allowed you to access the app locally at port 8000, which is mapped to port 80 in the container:
 
-`docker run --rm -it -p 9999:80 <my-app>`
+`docker run --rm -it -p 8000:80 <my-app>`
 
 ## New behavior
 
 Starting with .NET 8, if you map to port 80 in the container without explicitly setting the ASP.NET Core port used in the container, any attempt to connect to that mapped port will fail.
 
-For example, if you run the following command, you'd be unable to connect to the application locally using port 9999.
+For example, if you run the following command, you'd be unable to connect to the application locally using port 8000.
 
-`docker run --rm -it -p 9999:80 <my-app>`
+`docker run --rm -it -p 8000:80 <my-app>`
 
 Instead, change the command to use port 8080 within the container:
 
-`docker run --rm -it -p 9999:8080 <my-app>`
+`docker run --rm -it -p 8000:8080 <my-app>`
+
+You can see the difference in behavior in the following examples.
+
+Mapping port `80` (failure case):
+
+```bash
+$ docker run --rm -d -p 8000:80 mcr.microsoft.com/dotnet/samples:aspnetapp
+ba88b746bd7097e503f8ab6e5320c595640e242f6de4f734412944a0e2836acc
+$ curl http://localhost:8000/Environment
+curl: (56) Recv failure: Connection reset by peer
+$ docker kill ba88b746bd7097e503f8ab6e5320c595640e242f6de4f734412944a0e2836acc
+ba88b746bd7097e503f8ab6e5320c595640e242f6de4f734412944a0e2836acc
+```
+
+Mapping port `8080`:
+
+```bash
+$ docker run --rm -d -p 8000:8080 mcr.microsoft.com/dotnet/samples:aspnetapp
+74d866bdaa8a5a09e4a347bba17ced321d77a2524a0853294a123640bcc7f21d
+$ curl http://localhost:8000/Environment
+{"runtimeVersion":".NET 8.0.0-rc.1.23419.4","osVersion":"Alpine Linux v3.18","osArchitecture":"Arm64","user":"root","processorCount":4,"totalAvailableMemoryBytes":4123820032,"memoryLimit":0,"memoryUsage":30081024,"hostName":"74d866bdaa8a"}
+$ docker kill 74d866bdaa8a5a09e4a347bba17ced321d77a2524a0853294a123640bcc7f21d
+74d866bdaa8a5a09e4a347bba17ced321d77a2524a0853294a123640bcc7f21d
+```
+
+Mapping port `80` with `ASPNETCORE_HTTP_PORTS` set to port `80`:
+
+```bash
+$ docker run --rm -d -p 8000:80 -e ASPNETCORE_HTTP_PORTS=80 mcr.microsoft.com/dotnet/samples:aspnetapp 
+3cc86b4b3ea1a7303d83171c132b0645d4adf61d80131152936b01661ae82a09
+$ curl http://localhost:8000/Environment
+{"runtimeVersion":".NET 8.0.0-rc.1.23419.4","osVersion":"Alpine Linux v3.18","osArchitecture":"Arm64","user":"root","processorCount":4,"totalAvailableMemoryBytes":4123820032,"memoryLimit":0,"memoryUsage":95383552,"hostName":"3cc86b4b3ea1"}
+$ docker kill 3cc86b4b3ea1a7303d83171c132b0645d4adf61d80131152936b01661ae82a09
+3cc86b4b3ea1a7303d83171c132b0645d4adf61d80131152936b01661ae82a09
+```
 
```

Comment:
maybe add example with docker compose and point to https://docs.docker.com/compose/compose-file/05-services/#ports? 

in my case I had to edit the yml that consumed. I didn't immediately make the connection with docker run, because I wasn't familiar with docker compose.

Reply:
Good point. I added a link.